app/assets/javascripts/home.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```

Comment:
Using `rails g model` rather than `rails g scaffold` would solve this. `scaffold` creates a controller and views while `model` only creates the model, tests, and migrations